{
  "id": "PYSEC-2019-231",
  "details": "Google TensorFlow 1.6.x and earlier is affected by: Null Pointer Dereference. The type of exploitation is: context-dependent.",
  "aliases": [
    "CVE-2018-7576"
  ],
  "modified": "2021-12-09T06:35:11.756170Z",
  "published": "2019-04-23T21:29:00Z",
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2018-002.md"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "tensorflow-gpu",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/tensorflow-gpu"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "1.7.0rc0"
            }
          ]
        }
      ],
      "versions": [
        "0.12.0",
        "0.12.1",
        "1.0.0",
        "1.0.1",
        "1.1.0",
        "1.2.0",
        "1.2.1",
        "1.3.0",
        "1.4.0",
        "1.4.1",
        "1.5.0",
        "1.5.1",
        "1.6.0"
      ],
      "database_specific": {
        "source": "https://github.com/pypa/advisory-database/blob/main/vulns/tensorflow-gpu/PYSEC-2019-231.yaml"
      }
    }
  ],
  "schema_version": "1.3.0"
}